_posts/2016-08-02-should-i-use-a-spy-a-double-or-an-instance-double.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Should I use a spy, a double, or an instance_double?"
+date: 2016-08-02 15:03:00
+categories: ["rspec", "ruby"]
+author: "mauro-oto"
+---
+
+When writing tests for services, you may sometimes want to use mock objects
+instead of actual objects, which if you're using ActiveRecord,
```

Comment:
@mauro-oto I'd split this sentence into two: 

```
When writing tests, you may want to use mock objects instead of real objects. For example: When you use ActiveRecord and real objects, the test will use the database and slow down your suite.
```